### PR TITLE
Add Bullseye rootfs image for flashing Chrome OS on Chromebook devices

### PR DESCRIPTION
**Pull request by gctucker**
Add a rootfs configuration to flash Chrome OS images on Chromebook devices in LAVA.

### CODE REVIEW

**Comment by gctucker** [comment]
The GitHub check issue will be fixed by #1208

**Comment by gctucker** [comment]
Build completed on staging: https://storage.staging.kernelci.org/images/rootfs/debian/bullseye-cros-flash/20220523.0/amd64/

File: config/rootfs/debos/overlays/cros-flash/opt/chromeos/install-modules [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+install_modules()
+{
+    local mount_dir=$(basename "$IMAGE_MOUNTPOINT")
+    local root_path=$(dirname "$IMAGE_MOUNTPOINT")
+    local modules_tarball=$(basename "$MODULES_URL")
+
+    echo "Downloading modules from $MODULES_URL..."
+    mkdir -p "$IMAGE_MOUNTPOINT"
+    cd "$root_path"
+    wget --no-check-certificate "$MODULES_URL"
```
Comment (mgalka) [inline config/rootfs/debos/overlays/cros-flash/opt/chromeos/install-modules:14]: Do we need the --no-check-certificate if we have the ca-certificates package installed in the image?

**Review by mgalka — COMMENTED** [review]
small comment

**Comment by mgalka** [comment]
It looks that there is an issue with definitions order in the `rootfs-configs.yaml`

File: config/rootfs/debos/overlays/cros-flash/opt/chromeos/install-modules (hunk 1 above)
Comment (gctucker) [reply config/rootfs/debos/overlays/cros-flash/opt/chromeos/install-modules:14]: No, good point.  I'm dropping that now and also fixing other issues with flash and install-modules.

**Comment by gctucker** [comment]
Tested OK with manual LAVA job: https://lava.collabora.co.uk/scheduler/job/6445205

**Comment by gctucker** [comment]
> The GitHub check issue will be fixed by #1208

@mgalka About YAML validation errors.  Feel free to review that PR too ;)

**Comment by gctucker** [comment]
Tested OK with `install-modules`: https://lava.collabora.co.uk/scheduler/job/6445486

**Comment by gctucker** [comment]
Tested again on staging with 2nd image:
https://lava.collabora.co.uk/scheduler/job/6447043 (flash with Chrome OS image included)
https://lava.collabora.co.uk/scheduler/job/6447076 (install-modules and boot Chrome OS to login prompt)

**Pull request merged**